src/main/java/net/minecraftforge/event/server/ServerOverloadedEvent.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.event.server;
+
+import net.minecraft.server.MinecraftServer;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.server.ServerLifecycleHooks;
+
+/**
+ * ServerOverloadedEvent is fired when the server is running behind ticking schedule and can't keep up.
+ * You should avoid doing any heavy work in this event as doing so may cause the server watchdog to timeout and crash the server.<br>
+ * <br>
+ * Please do NOT use this event as a trigger for disabling Vanilla features - this may cause unexpected behaviour for players and
+ * hard to reproduce compatibility issues with other mods that expect consistent Vanilla behaviour.<br>
+ * <br>
+ * Here's a couple of examples of intended usage of this event:
+ * <ol>
+ *     <li>to log data that could be useful for diagnosing why the server is overloaded</li>
+ *     <li>for sending automated alerts to server staff outside of the game</li>
+ * </ol>
+ * <br>
+ * This event is fired via {@link ServerLifecycleHooks#onServerOverloaded(MinecraftServer, long, long, long)},
+ * which is executed from {@link MinecraftServer#runServer()} <br>
+ * <br>
+ * This event is not {@link Cancelable}.<br>
+ * <br>
+ * This event does not have a result. {@link HasResult}<br>
+ * <br>
+ * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+ */
```

Comment:
I suggest some improvements to this javadoc:

- Using the somewhat standardized format found currently in the client events.
- Using paragraph tags to mark paragraphs (except the first, which is implicitly a paragraph) instead of the line break tag.
- Using an unordered list instead of an ordered list, as the order of the list items doesn't matter.
- Removing the part about where the event is fired from, since that is information not needed by consumers.

```suggestion
/**
 * Fired when the server is running behind ticking schedule and can't keep up. Heavy work should be avoided in listeners
 * for this event, as doing so may cause the {@link net.minecraft.server.dedicated.ServerWatchdog} to timeout and crash
 * the server.
 *
 * <p>Please <strong>do not use this event for disabling vanilla features</strong> - this may cause unexpected behavior
 * for players and hard-to-reproduce compatibility issues with other mods that expect consistent vanilla behavior.</p>
 * 
 * <p>Here are some examples of intended usages of this event:</p>
 * <ul>
 *     <li>to log data that could be useful for diagnosing why the server is overloaded</li>
 *     <li>for sending automated alerts to server staff outside of the game and console</li>
 * </ul>
 *
 * <p>This event is not {@linkplain Cancelable cancellable}, and does not {@linkplain HasResult have a result}.</p>
 *
 * <p>This event is fired on the {@linkplain MinecraftForge#EVENT_BUS main Forge event bus},
 * only on the {@linkplain LogicalSide#SERVER logical server}.</p>
 */
```